docs-content/installing.html.md.erb

```diff
@@ -69,6 +69,39 @@ $ cf curl /v2/info
   * **Username for Reporting Custom Dashboard**: Username of an account with custom dashboard creation permissions.
   * **Password**: Password for the above Account.
 
+## <a id='configure-proxy-settings'></a> Configuring Proxy Settings for the AppDynamics Platform Monitoring Tile
+
+If you're using a Proxy Server and SaaS controller, the two PCF apps deployed by the AppDynamics Platform Monitoring for PCF tile, AppDynamics Nozzle and the AppDynamics KPI Dashboard, require that you set environment variables to define the proxy host and port so the apps can communicate with the configured SaaS controller. 
+
+The following instructions assume the use of the Cloud Foundry CLI. You could also set the environment variables from the Apps Manager as well.
```

Comment:
This sentence can be removed: **You could also set the environment variables from the Apps Manager as well.**